README.md

```diff
@@ -236,13 +236,14 @@ Bit     | Name     | Description
    26   |          | encodes invts and flushts, see below
    25   | crp (?)  |
    20   | type     | type is 1 for SF2
+  5:0   | pred     | `pred` num
```

Comment:
Are you sure that `pred` is 6 bits wide? There are only 32 predicate registers.

Reply:
Ouch. It should be 4:0